metaci/build/models.py

```diff
@@ -127,7 +127,7 @@ class Build(models.Model):
     org_instance = models.ForeignKey('cumulusci.ScratchOrgInstance',
                                      related_name='builds', null=True, blank=True, on_delete=models.CASCADE)
     schedule = models.ForeignKey('plan.PlanSchedule', related_name='builds',
-                                 null=True, blank=True, on_delete=models.CASCADE)
+                                 null=True, blank=True, on_delete=models.SET_NULL)
```

Comment:
You'll need to generate a migration for this.

Maybe we should have a test that runs a makemigrations dry run and breaks if it did anything.

Reply:
Yes it does generate a migration but based on the Django documentation it is my belief that the migration doesn't really do anything. It will end up re-creating the same ForeignKey with the same SQL but different Python behaviour. I guess I'll create it now to avoid some future programmer confusion about where it came from...

Can you remind me though: will deploying to prod run the migration automatically?

Reply:
3a6522f29309aac7f53d6a5085e62bff6769b711

Reply:
I don't think the migration will actually change anything in the database, but it's still needed for the internal bookkeeping of the migration system. If the behavior of a subsequent migration would be affected by this change in the field setting, the migration system needs to know that this change happened first, I guess.

Yes, deploying will run the migration. That's what the `release` line in Procfile is for.